src/action-creators/ModuleInfoDocs.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Here's documentation for all action-creators.
+ *
+ * @packageDocumentation
+ * @module action-creators
+ * @preferred
+ */
```

Comment:
What's the purpose of this file, and why is it a js file? I don't like having dummy files for the docs.

Reply:
I've added these files as illustration that we are able to add special description for each module.
Text from these files is displayed above the list of functions inside a module.
![Screenshot from 2020-06-17 21-04-13](https://user-images.githubusercontent.com/32064251/84909243-a58e8d80-b0df-11ea-8f5b-38d80a64a489.png)


We can remove it, if we doesn't need it


![Screenshot from 2020-06-17 21-03-49](https://user-images.githubusercontent.com/32064251/84909303-b4754000-b0df-11ea-869f-7444a60fc8e9.png)

Or we can move such files to another directory. See a5bcdc3

Also typedoc supports markdown, so we can write something like this:


![Screenshot from 2020-06-17 21-13-26](https://user-images.githubusercontent.com/32064251/84909618-1d5cb800-b0e0-11ea-934b-311763dd32b3.png)

Reply:
> I've added these files as illustration that we are able to add special description for each module.

Yes, I think that would be useful to have a description for each module. It's just that storing it in an arbitrary js file that is only used for documentation seems less than ideal.

> Or we can move such files to another directory.

Good to know, although I think this is worse, as they should definitely be colocated.

Reply:
The logical place for the module description to go would be the `index.js` file in each module folder, however TypeDoc didn't seem to pick up the package documentation there when I added it. Can you look into this to see if it's possible?

Reply:
Yes, you are right! TypeDoc didn't pick index.js files because i excluded them in config file :)
I did it because TypeDoc creates References for re-exported files.
![Screenshot from 2020-06-19 16-36-41](https://user-images.githubusercontent.com/32064251/85118975-42216e80-b24b-11ea-8c66-4a4bb2ce2671.png)
I added `typedoc-plugin-exclude-references` to fix this, now any description could be placed in index.js files

Commit: 395942c

Reply:
Excellent!